draft-ietf-rats-msg-wrap.md

```diff
@@ -650,11 +650,10 @@ The following subsections provide further elaboration on these points, particula
 
 ## CMW Protection
 
-CMW itself does not provide any mechanisms for authenticity, integrity protection, or confidentiality.
+CMW Records, Tags and Collections alone don't offer authenticity, integrity protection, or confidentiality.
 It is the responsibility of the designer for each use case to determine the necessary security properties and implement them accordingly.
-In some scenarios, a secure channel (e.g., via TLS {{?BCP195}}) or object-level security (e.g., using JWT {{?BCP225}}) may be sufficient, but this is not always the case.
-
-When a CMW is used to carry Evidence for composite or layered attestation of a single device, all components within the CMW must be cryptographically bound to prevent an attacker from replacing Evidence from a compromised device with that from a non-compromised device.
+In some scenarios, for instance, if CMW encapsulates Reference Values or Endorsements, a secure channel (e.g., via TLS {{?BCP195}}) or object-level security (e.g., using JWT {{?BCP225}}) may suffice, but this is not always the case.
+When a CMW is used to encapsulate Evidence for composite or layered attestation of a single device, all components within the CMW must be cryptographically bound to prevent an attacker from replacing Evidence from a compromised device with that from a non-compromised device.
```

Comment:
Is this a normative MUST? I don't think that's necessarily what we're going to see, even if it is unfortunate. The binding is up to the protocol, and not this envelope format.
```suggestion
When a CMW is used to encapsulate Evidence for composite or layered attestation of a single device, all components within the CMW must be cryptographically bound together to prevent an attacker from replacing Evidence from a compromised device with that from a non-compromised device.
```